Simulation/Convert_StateVector.m

```diff
@@ -0,0 +1,11 @@
+%Convert_StateVector
+
+SimData.Plane.Mass = 1
+SimData.Plane.AeroRefArea = 1
+SimData.Plane.Cd = 1
+SimData.Plane.Cl = 1
+SimData.Plane.AoA = 1
+SimData.StateVector.Position = [0,0]
+SimData.StateVector.Velocity = [0,0]
+SimData.StateVector.Orientation = [Pitch]
+SimData.Time = 0
```

Comment:
This file does not seem to be used anywhere so I think it can be removed.

Reply:
Removed